docs/developer-guide/plugins.md

```diff
@@ -9,25 +9,25 @@ Each plugin is contained within its own JAR file, located according to the confi
 ## Configuration example
 
     plugins:
-      active: rewrite
+      active: addheaders
       all:
-        - name: rewrite
+        addheaders:
           factory:
-            class: "com.hotels.styx.RewritePluginFactory"
+            class: "com.hotels.styx.ExamplePluginFactory"
             classPath: "<path-to-plugin>/plugin-examples-1.0-SNAPSHOT.jar"
           config:
-            oldUri: "/olduri"
-            newUri: "/newuri"
+            requestHeaderValue: "requestheader"
+            responseHeaderValue: "responseheader"
       
 ### Configuration example explanation
 
 * **active** contains a comma-separated list of plugin names. It is a convenience feature used to switch a plugin on and off without needing to remove its entire set of configuration.
 * **all** contains a list of configuration for all the plugins deployed with the Styx instance. Inside each list item:
-    * **name** is the name of the plugin
+    * **addheaders** is the name of the plugin
     * **factory** configures a factory object that can produce the plugin:
         * **class** contains the name of the factory class, which must extend `com.hotels.styx.api.plugins.spi.PluginFactory`
         * **classPath** provides the location of the JAR file
-    * **config** contains custom configuration for the particular plugin. Its value can be of any type. See below for how to write a custom config java class.
+    * **config** contains custom configuration for the particular plugin. Its value can be of any type. See below how to write a custom config java class.
 
 When Styx starts, it sets up the HTTP interceptor chain as follows:
 
```

Comment:
There is a chapter called `Custom Configuration` that is somewhat out of place. I would move that further down under the `Developing Plugin` chapter.